### DISCUSSION TITLE
organizzazione fisica - 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#80] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/40fada77-1fb7-41b5-a688-816b0e10970f)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-22 155555](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/19da8562-d554-4e2e-8ee0-e112ab2a72c2)
![Immagine 2024-01-22 155607](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/113d91da-4f0c-47a5-800b-84e10d90b560)
![Immagine 2024-01-22 155619](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/fdcc252c-a20b-461f-900a-c83c4e5973c7)

**Reply by 7Andrei** [reply]
Mi viene tutto uguale, tranne il numero totale di blocchi per bucket e bucket directory che mi viene 375002, dato che ci sono i 2 blocchi della directory.

**Comment by sprigatita** [comment]
![IMG_0670](https://github.com/user-attachments/assets/87fc363b-9dd9-40ce-8875-35de075cdd18)
